src/providers/idp/idp_id_eval.c

```diff
@@ -81,7 +81,11 @@ errno_t store_json_user(struct idp_id_ctx *idp_id_ctx, json_t *user,
         ret = EIO;
         goto done;
     }
-    gid = uid;
+    if (dom->mpg_mode != MPG_DISABLED) {
+        gid = 0;
```

Comment:
Why?
If 'mpg_mode == MPG_ENABLED' and so '!= MPG_DISABLED', why 'gid = 0'?
Man page says 'MPG_ENABLED' (i.e. 'true') means "Create user's private group unconditionally from user's UID number."

Reply:
This is for `sysdb_add_user()` https://github.com/SSSD/sssd/blob/master/src/db/sysdb_ops.c#L1863

Reply:
That's somewhat confusing.
Practically there is no difference what MPG setting is, anyway it will be 'gid == uid'.